crates/sui-open-rpc/spec/openrpc.json

```diff
@@ -3151,14 +3151,12 @@
           "result": {
             "name": "Result",
             "value": {
-              "id": {
-                "id": "0x6d907beaa3a49db57bdfdb3557e6d405cbf01c293a53e01457d65e92b5d8dd68"
-              },
               "decimals": 9,
               "name": "Usdc",
               "symbol": "USDC",
               "description": "Stable coin.",
-              "icon_url": null
+              "iconUrl": null,
+              "id": "0x6d907beaa3a49db57bdfdb3557e6d405cbf01c293a53e01457d65e92b5d8dd68"
```

Comment:
This is the other meaningful change -- changing how the `id` is represented and typo on `iconUrl`.